solutions/observability/streams/streams.md

```diff
@@ -1,47 +1,84 @@
+
+### Wired streams
+```{applies_to}
+stack: preview 9.2
+serverless: preview
+```
 
-A Stream directly corresponds to an {{es}} data stream (for example, `logs-myapp-default`). Operations performed in the Streams UI configure that specific data stream.
+Wired streams data is sent directly to a single endpoint, from which you can route data into child streams based on [partitioning](./management/partitioning.md) set up manually or with the help of AI suggestions.
+
+Wired streams:
+- Allow you to organize streams in a parent-child hierarchy.
+- Let child streams automatically inherit mappings, lifecycle settings, and processors.
+- Send configuration changes through the hierarchy to keep streams consistent.
+
+For more information, refer to [sending data to wired streams](./wired-streams.md).
 
 ## Required permissions
 
-Streams requires the following Elastic Cloud Serverless roles:
+Streams requires the following permissions:
+
+::::{tab-set}
+
+:::{tab-item} Serverless
+Streams requires these Elastic Cloud Serverless roles:
 
 - Admin: ability to manage all Streams.
 - Editor/Viewer: limited access, unable to perform all actions.
 
-## Access the Streams UI
+:::
 
-In {{obs-serverless}}, Streams is automatically available.
+:::{tab-item} Stack
+To manage all Streams, you need the following permissions:
+
+- Cluster permissions: `manage_index_templates`, `manage_ingest_pipelines`, `manage_pipeline`, `read_pipeline`
+- Data stream level permissions: `read`, `write`, `create`, `manage`, `monitor`, `manage_data_stream_lifecycle`, `read_failure_store`, `manage_failure_store`, `manage_ilm`.
+
+To view streams, you need the following permissions:
+- Data stream level: `read`, v`iew_index_metadata`, `monitor`
+
+:::
 
-In {{stack}} version 9.1 and later, you can enable Streams in the {{observability}} Advanced Settings. To do this:
+::::
 
-1. Go to the **Advanced Settings** page in the navigation menu or by searching fo, or search for "Advanced Settings" in the [global search field](/explore-analyze/find-and-organize/find-apps-and-objects.md).
-1. Enable **Streams UI** under **Observability**.
+## Access Streams
 
-In {{serverless-short}} or after enabling Streams in {{stack}}, access the UI in one of the following ways:
+Open Streams from the following places in {{kib}}:
 
 - Select **Streams** from the navigation menu or use the [global search field](../../../explore-analyze/find-and-organize/find-apps-and-objects.md).
 
-- From **Discover**, expand a document's details flyout and select **Stream** or an action associated with the document's data stream. Streams will open filtered to only the selected stream. This only works for documents stored in a data stream.
+- Open the data stream for a specific document from **Discover**. To do this, expand the details flyout for a document that's stored in a data stream, and select **Stream** or an action associated with the document's data stream. Streams will open filtered to the selected data stream.
 
```

Comment:
Should we also link to the API documentation somewhere here (with a note that it's tech preview)? Wdyt @LucaWintergerst ?

Reply:
Yes, that would be ideal